### PR TITLE
feat(AddressForm.tsx): add appartment field

### PR DESCRIPTION
**Pull request by RoeiRom**

### CODE REVIEW

**Review by Noam6421 — DISMISSED** [review]

**Comment by y-golde** [comment]
what about the excel? , It still says floor :\

File: client/src/commons/Forms/AddressForm/AddressForm.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ import useStyles from './AddressFormStyles';
 const CITY_LABEL = 'עיר';
 const STREET_LABEL = 'רחוב';
 const FLOOR_LABEL = 'קומה';
+const APPARTMENT_LABEL = 'דירה';
```
Comment (y-golde) [inline client/src/commons/Forms/AddressForm/AddressForm.tsx:19]: apartement is spelled with 1 p in the server and 2 ps in here

File: client/src/commons/Forms/AddressForm/AddressForm.tsx [hunk 2]
```diff
@@ -26,6 +27,7 @@ const AddressForm: React.FC<Props> = ({
     cityField, 
     streetField,
     floorField, 
+    appartmentField,
```
Comment (roymaniv) [inline client/src/commons/Forms/AddressForm/AddressForm.tsx:30]: change spelling here as well

Comment (roymaniv) [inline client/src/commons/Forms/AddressForm/AddressForm.tsx:30]: change spelling here as well

File: client/src/commons/Forms/AddressForm/AddressForm.tsx [hunk 3]
```diff
@@ -44,8 +46,9 @@ const AddressForm: React.FC<Props> = ({
         }
     }, [cityWatcher]);
 
-    const houseNumberFieldNameSplitted = houseNumberField.name.split('.');
-    const floorFieldNameSplitted = floorField.name.split('.');
+    const houseNumberFieldNameSplitted = houseNumberField?.name.split('.');
+    const floorFieldNameSplitted = floorField?.name.split('.');
+    const appartmentFieldNameSplitted = appartmentField?.name.split('.');
```
Comment (roymaniv) [inline client/src/commons/Forms/AddressForm/AddressForm.tsx:51]: and here

**Review by y-golde — APPROVED** [review]

**Review by Noam6421 — APPROVED** [review]

**Pull request merged**